tests/run_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+pushd tests
+
+function finish {
+   popd
+}
+trap finish EXIT
+
+pipenv install
+cargo build --verbose $@
+pipenv run pytest
```

Comment:
When this runs, [this code](https://github.com/RustPython/RustPython/blob/master/tests/test_snippets.py#L81) will probably still force a release build, right?

Reply:
Good catch. I've added a `RUST_RELEASE` environment variable to handle this case